.github/workflows/build.yml

```diff
@@ -0,0 +1,63 @@
+name: Prime Data Hub
+
+on:
+  pull_request:
+    branches: [ master, production ]
+
+defaults:
+  run:
+    working-directory: prime-router
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set Environment Variables
```

Comment:
This chunk will grab all of the secrets (depending on environment) and set them as environment variables during the build.